City-of-Helsinki/hitas
# Fixed the navigation bar to use React Router

# Hitas Pull Request

# Description

Fixed the navigation bar to use React Router and made corresponding changes to routes and authentication.

## Pull request checklist

Check the boxes for each DoD item that has been completed:

- **Testing**
- **Database**
- **Documentation**

## Test plan

- Check that RTK Query caching works now as expected: 
  - Navigate through the app using the navigation bar links and check that clicking a navi link does not cause new request if the same data has already been requested.
- Check that authorization still protects the content as expected

## Tickets

This pull request resolves all or part of the following ticket(s): HT-569
